admin/tabs/configs/differ/ConfigDifferModel.js

```diff
@@ -82,10 +84,12 @@ export class ConfigDifferModel  {
     }
 
     async loadAsync() {
+        const remoteHost = `${trimEnd(this.remoteHost, '/') + (XH.baseUrl == '/api/' ? '/api' : '/')}`;
```

Comment:
Realizing this is tricky to get really correct as it relates to the remote host and knowing what its baseUrl configuration (and therefore proxy configuration) might be, which isn't easily knowable.

However I think it makes a lot of sense to assume that a remote host has the same baseUrl value as the current host - i.e. that they share the same convention - so checking XH.baseUrl here does make sense to me and seems expedient.

Rather than hard-code the check to look for /api/ however, can we instead check if it's a relative URL and then append it, whatever it is?  That way if for some reason you had a collection of app instances that had decided they didn't want to use our /api/ convention and wanted to standardize on something like /grailsBackEnd/ this code could still work.

Reply:
Great. I had a comment here to a similar effect yesterday (which i deleted after review with Lee). In addition, this line prompted the idea for XH.isLocalDevelopment. I guess it wouldn't be used here under this plan, but still might be useful to have in general?